unmount_linux_test.go

```diff
@@ -0,0 +1,31 @@
+package fuse
+
+import (
+	"errors"
+	"testing"
+)
+
+func Test_umountExpectCustomError(t *testing.T) {
+	dir := "/dev/fd/42"
+	fuserunmountMock = func(string) error {
```

Comment:
Instead of mocking the function, would it be possible to use `t.Setenv("PATH", "")` here and thereby make `exec.LookPath` return an error? That way, we don’t need to change the non-test code only for testing.